src/middleware.ts

```diff
@@ -60,7 +60,7 @@ export const expressAggressiveCache = (options?: Options) => {
       const originalWrite: any = res.write;
       const originalEnd: any = res.end;
       const chunkQueue = new Queue();
-      const normalizedUrl = defaultGetCacheKey({ req, res });
+      const normalizedUrl = defaultGetCacheKey({ req });
```

Comment:
As we discussed, we should remove the http verb from the cache key and call this a `normalizedPath` instead.